script/deploy.sh

```diff
@@ -11,14 +11,9 @@ fi
 git config --global user.email "${TRAEFIKER_EMAIL}"
 git config --global user.name "Traefiker"
 
-# load ssh key
-eval "$(ssh-agent -s)"
-chmod 600 /home/semaphore/.ssh/traefiker_rsa
-ssh-add /home/semaphore/.ssh/traefiker_rsa
-
 # update traefik-library-image repo (official Docker image)
 echo "Updating traefik-library-imag repo..."
-git clone git@github.com:traefik/traefik-library-image.git
+git clone "https://${GITHUB_TOKEN}@github.com/traefik/traefik-library-image.git"
```

Comment:
Don't use token an use Github secret to store SSH key to connect to https://github.com/traefik/traefik-library-image